src/App.jsx

```diff
@@ -10,19 +14,38 @@ class App extends React.Component {
     selectedUserId: 0,
   };
 
+  componentDidMount() {
+    getTodos()
+      .then(todos => 
+        this.setState({ todos }))
+  };
+
+  clearSelectedUser = () => {
+    this.setState({selectedUserId: 0})
+  };
+
   render() {
     const { todos, selectedUserId } = this.state;
 
     return (
       <div className="App">
         <div className="App__sidebar">
-          <TodoList todos={todos} />
+          <TodoList
+            todos={todos}
+            onUserIdSelected={userId => {
+              this.setState({selectedUserId: userId})
```

Comment:
For better reading you can create a function like `selectUser` and hust pass it here:
`onUserIdSelected={selectUser}`